ts/screens/profile/components/ShareDataComponent.tsx

```diff
@@ -87,16 +89,32 @@ export const ShareDataComponent = (): React.ReactElement => {
             )}
           </Label>
         </Body>
-        <Link onPress={() => openWebUrl(ioSuppliersUrl)}>
+        <Link
+          onPress={() => openWebUrl(ioSuppliersUrl)}
+          accessibilityRole="link"
+        >
           {I18n.t("profile.main.privacy.shareData.screen.gdpr.cta")}
         </Link>
       </InfoBox>
       <View spacer={true} />
-      <Body>
+      <Body
+        accessibilityRole="link"
+        onPress={() => {
+          if (isScreenReaderEnabled) {
+            openWebUrl(privacyUrl);
+          }
+        }}
+      >
```

Comment:
This solution LGTM but perhaps it may be acceptable to have this behavior even for those who do not have a screen reader, just to avoid to maintain two conditional paths (tap on the body and open the TOS for everyone). What you, @FrancescaMariaTIberti and @thisisjp think about this, could it be a simplification?

Reply:
@fabriziofff you mean tapping on the whole paragraph and you get the bottom sheet, right? Okay for me 👌

Reply:
Yes, I meant that! 
In this case if @andrea-favaro agree too, I suggest to avoid the branching based on the screen reader and having only common behaviour aka -> Tapping on the whole paragraph.